### PR TITLE
Update Login Servlet and New User Servlet

### PR DESCRIPTION
**Pull request by winniewan**
Add userId and isBusiness variables so that LoginServlet can pass these information to the client side. 

Update New User Servlet unit tests implementation by removing user service and datastore mocks. 
New User Servlet will add default values to datastore for new users that are signing in to webpage.

### CODE REVIEW

File: capstone/src/main/java/com/google/sps/data/User.java [hunk 1]
```diff
@@ -8,14 +8,22 @@
   // The logged-in or logged-out URL links.
   private final String url;
 
+  // The logged-in or logged-out URL links.
```
Comment (alygerharter) [inline capstone/src/main/java/com/google/sps/data/User.java:11]: update this javadoc

File: capstone/src/main/java/com/google/sps/data/User.java [hunk 2]
```diff
@@ -8,14 +8,22 @@
...
+  private final String userId;
+
+  // The logged-in or logged-out URL links.
```
Comment (alygerharter) [inline capstone/src/main/java/com/google/sps/data/User.java:14]: update this

File: capstone/src/main/java/com/google/sps/data/User.java (hunk 1 above)
Comment (winniewan) [reply capstone/src/main/java/com/google/sps/data/User.java:11]: Oops, good catch!

File: capstone/src/main/java/com/google/sps/data/User.java (hunk 2 above)
Comment (winniewan) [reply capstone/src/main/java/com/google/sps/data/User.java]: done

File: capstone/src/main/java/com/google/sps/servlets/authentication/LoginServlet.java [hunk 3]
```diff
@@ -18,24 +24,38 @@
...
 
-  public LoginServlet() {}
+  private UserService userService = UserServiceFactory.getUserService();
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-  public LoginServlet(UserService userService) {
-    this.userService = userService;
-  }
+  private String loginUrl = userService.createLoginURL("/check_new_user");
+  private String logoutUrl = "/logout";
```
Comment (lukasberglund) [inline capstone/src/main/java/com/google/sps/servlets/authentication/LoginServlet.java:34]: Could we make these `final`?

Comment (winniewan) [reply capstone/src/main/java/com/google/sps/servlets/authentication/LoginServlet.java]: Yep

File: capstone/src/main/java/com/google/sps/servlets/authentication/LoginServlet.java [hunk 4]
```diff
@@ -23,15 +23,14 @@
...
   private static final String IS_BUSINESS = "isBusiness";
   private static final String SUPPORT_PROPERTY = "support";
-
-  private UserService userService = UserServiceFactory.getUserService();
-  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-
-  private String loginUrl = userService.createLoginURL("/check_new_user");
-  private String logoutUrl = "/logout";
+  private final String loginUrl = userService.createLoginURL("/check_new_user");
+  private final String logoutUrl = "/logout";
```
Comment (lukasberglund) [inline capstone/src/main/java/com/google/sps/servlets/authentication/LoginServlet.java:33]:
```suggestion
  private final String LOGIN_URL = userService.createLoginURL("/check_new_user");
  private final String LOGOUT_URL = "/logout";
```

Comment (lukasberglund) [reply capstone/src/main/java/com/google/sps/servlets/authentication/LoginServlet.java:33]: Sorry for this double nit!!

Comment (winniewan) [reply capstone/src/main/java/com/google/sps/servlets/authentication/LoginServlet.java]: done

**Pull request merged**